source/fuzz/fuzzer_context.cpp

```diff
@@ -65,6 +65,7 @@ const std::pair<uint32_t, uint32_t> kChanceOfOutliningFunction = {10, 90};
 const std::pair<uint32_t, uint32_t> kChanceOfPermutingParameters = {30, 90};
 const std::pair<uint32_t, uint32_t> kChanceOfPermutingPhiOperands = {30, 90};
 const std::pair<uint32_t, uint32_t> kChanceOfPushingIdThroughVariable = {5, 50};
+const std::pair<uint32_t, uint32_t> kChanceOfRemovingParameters = {30, 70};
```

Comment:
If we change the name of the fuzzer pass (see below) we should change this name (and related names in these files) accordingly.

Reply:
Fixed in cc488d5.